include/item.h

```diff
@@ -10,7 +10,7 @@ typedef void (*ItemUseFunc)(u8);
 struct Item
 {
     u8 name[ITEM_NAME_LENGTH];
-    u16 price;
+    u32 price;
     u8 holdEffect;
     u8 holdEffectParam;
     const u8 *description;
```

Comment:
We want to probably reorder this struct to avoid unnecessary padding.

Reply:
i.e.
```c
struct Item
{
    u32 price;
    const u8 *description;
    ItemUseFunc fieldUseFunc;
    u16 secondaryId;
    u8 name[ITEM_NAME_LENGTH];
    u8 holdEffect;
    u8 holdEffectParam;
    u8 importance;
    u8 pocket;
    u8 type;
    u8 battleUsage;
    u8 flingPower;
};
```
would save us 8 bytes of padding.

Reply:
This shouldn't break anything correct?

Reply:
Ok yea it doesn't, so I added it.